docs/schedule/examples/fares-v1.md

```diff
@@ -6,6 +6,9 @@
 
 <hr>
 
+Composed of [fare_attributes.txt](../../reference/#fare_attributestxt) and [fare_rules.txt](../../reference/#fare_rulestxt), Fares v1 is the current official method for describing fare information in GTFS. However the two files are limited in the breadth of factors they can efficiently describe, and are ambiguous to implement. 
+[Fares v2](../../examples/fares-v2/) is an extension project under active development, and it aims to address the limitations of Fares v1.
```

Comment:
``Fares v1 is the current official method for describing fare information in GTFS`` - is this accurate given the base implementation of Fares v2 being adopted? All other explanation is clear, but this confuses me. @isabelle-dr

Reply:
Good point, since we have Fares v2 now the two options are available. I'll modify accordingly

Reply:
Attempt to solve this in [this commit](https://github.com/MobilityData/gtfs.org/pull/58/commits/d4559ebaea43812c1b06ab976544f96fb30ee41a)

Reply:
Much clearer, thank you!